content/08-mongodb/11-mongodb-atlas-setup.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: 'Introduction to provisioning MongoDB Atlas'
+metaTitle: "MongoDB Atlas Setup | Prisma's Data Guide"
+metaDescription: "In this guide, we'll cover provisioning MongoDB Atlas."
+metaImage: '/content/mongodb/header.png'
+authors: ['alexemerich']
+---
+## Introduction
+
+MongoDB Atlas is MongoDB's cloud database service. It offers data distribution and mobility across major cloud providers AWS, Azure, and Google Gloud. It is popular for its built-in automation for resource and workload optimization. 
+
+In this guide, we are going to walk through the steps to provision a MongoDB Atlas cluster from the browser. We'll walk through all the options to configuring a cluster up until deploy.
+
+**Note:** Some instructions and images are subject to change since the writing of this guide in February 2022. We'll do our best to keep as up to date as possible as changes occur.
+
+<PrismaOutlinks>
+
+If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) with MongoDB, you can use the [MongoDB connector (currently in preview)](https://www.prisma.io/docs/concepts/database-connectors/mongodb) to connect and manage your data.
+
+</PrismaOutlinks>
+
+## How to provision a MongoDB Atlas cluster
+
+To begin setting up your new Atlas cluster, you will navigate to the [Atlas registration page](https://www.mongodb.com/cloud/atlas/register) and create a new account. After successful registration, you will be directed to a page where you will create your first cluster.
+
+If you already have a MongoDB account registered, you can login from the registration page. Once logged in, you will want to click on the green "+ Create" button pictured below. This will take you to the same page for creating a cluster as if you were creating a cluster for the first time.
+
+![Create Cluster](/content/mdb-atlas-setup/create-cluster-button.png)
+
+The first decision you will have to make is what kind of cluster you would like to configure. At the time of this writing, MongoDB offers three options: Serverless (Preview), Dedicated, and Shared. 
+
+![Cluster Type](/content/mdb-atlas-setup/cluster-type.png)
+
+The serverless instance is suitable for serverless applications with variable or infrequent traffic. There is minimal configuration and resources scale to meet your workload with cost matching your actual usage.
```

Comment:
```suggestion
The serverless instance is suitable for serverless applications with variable or infrequent traffic. There is minimal configuration and resources scale to meet your workload with the cost matching your actual usage.
```

Just a small suggestion to improve readability for me.  Without the "the", my brain tried to make "cost matching" a phrase similar to like retail "price matching" and I got myself turned around a bit.